packages/react/src/themes/teams/components/Chat/chatMessageVariables.ts

```diff
@@ -26,18 +26,18 @@ export default (siteVars): ChatMessageVariables => ({
   actionMenuPositionRight: pxToRem(5),
   actionMenuPositionTop: pxToRem(-30),
   width: '100%',
-  backgroundColor: siteVars.white,
+  backgroundColor: siteVars.colors.white,
   backgroundColorMine: '#E5E5F1',
   borderRadius: pxToRem(3),
   color: 'rgb(64, 64, 64)',
   padding: pxToRem(16),
   authorMarginRight: pxToRem(12),
   headerMarginBottom: pxToRem(2),
-  contentFocusOutlineColor: siteVars.brand,
+  contentFocusOutlineColor: siteVars.colors.primary[500],
   border: 'none',
   badgeShadow: siteVars.shadowLevel1Darker,
   isImportant: false,
   hasMention: false,
-  hasMentionColor: siteVars.orange04,
-  isImportantColor: siteVars.red,
+  hasMentionColor: siteVars.naturalColors.darkOrange[400],
```

Comment:
```suggestion
  hasMentionColor: siteVars.colors.darkOrange[400],
```

Reply:
Actually I think this was producing TS errors, that's why I have the `naturalColors`. Let me check and will confirm

Reply:
The problem is that the `NaturalColorsStrict` doesn't contain this color.

Reply:
Hm, it seems that it's working.. Never mind will change it and report back if I have an issue with that.

Reply:
Created issue with proposed fixed: https://github.com/stardust-ui/react/issues/898